packages/web-components/fast-ssr/package.json

```
@@ -44,6 +49,7 @@
     "tslib": "^2.4.0"
   },
   "devDependencies": {
+    "@microsoft/api-extractor": "7.23.0",
```

I'm not sure you can make this change without potential issues with two versions, but I pushed a PR (not in FE2 though) to update API Extractor to the published version which includes the fix we need for typescript 4.7.0 (beta). Whether we do it here or not, we'll probably want to update this prior to the merge of FE2.
```suggestion
    "@microsoft/api-extractor": "7.23.1",
 ```

---

I'll try it and see

---

Seems to, update pushed.